flytekit/core/type_engine.py

```diff
@@ -126,6 +126,10 @@ def to_python_value(self, ctx: FlyteContext, lv: Literal, expected_python_type:
             f"Conversion to python value expected type {expected_python_type} from literal not implemented"
         )
 
+    @abstractmethod
+    def to_html(self, ctx: FlyteContext, python_val: T):
```

Comment:
is to_html, not going to return a div? it can be a simple string?
then maybe html string

Reply:
can we write docs for this